tests/testapp/forms.py

```diff
@@ -174,3 +174,25 @@ class Meta:
         widgets = {
             'genres': GenreSelect2TagWidget
         }
+
+
+class AddressChainedSelect2WidgetForm(forms.Form):
+    country = forms.ModelChoiceField(
+        queryset=Country.objects.all(),
+        label=u"Country",
+        widget=ModelSelect2Widget(
+            search_fields=['name__icontains'],
+            max_results=500,
+            dependent_fields={'city': 'cities'},
```

Comment:
why or which?

If You about 'dependent_fields={'city': 'cities'},', then:
this is for demonstration (and test) interdependent fields.